keyboards/synthlabs/065/config.h

```diff
@@ -0,0 +1,61 @@
+// Copyright 2023 Aaron Hong (@hongaaronc)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+/* Backlighting */
+#define RGB_MATRIX_LED_COUNT 83
+#define RGB_MATRIX_KEYPRESSES
+#define RGB_MATRIX_FRAMEBUFFER_EFFECTS
+#define RGB_DISABLE_WHEN_USB_SUSPENDED
+#define RGB_DI_PIN GP18
```

Comment:
```suggestion
#define WS2812_DI_PIN GP18
```

Reply:
I tried applying your suggestion, but ended up reverting it since it seems like the change causes a compilation error.

Here's what I'm running into: https://pastebin.com/GQKFX3Hf

I've also tried removing `WS2812_DRIVER = vendor` from `rules.mk`, but this also seems to run into a different error: https://pastebin.com/rhk3i0Ya

Should I just leave this as-is?

Reply:
I'm an idiot 🤦‍♂️

Turns out I just hadn't synced the latest core changes into my fork repo in a while which is why it wasn't working.

Anyway, I ended up replacing the WS2812 pin and driver declarations from `config.h` and `rules.mk` with a declaration in `info.json`, since that seems to be cleaner (edd1e9c).